config/routes.rb

```diff
@@ -14,9 +12,10 @@
 get "user/login"
 get "user/sign_up"
 get "employee/employee"
-get "laptop/laptop"
 get "employees/index"
 
+get 'assets/laptops', to: 'assets#laptops', as: 'laptops'
```

Comment:
I know this is going to change later to the search term parameters thing in the URL.

But for now I think it should be **/asset-directory/laptops** just because as a user you can see that it is another level deeper from /asset-directory